service-front/module/Application/src/Helpers/FormProcessorHelper.php

```diff
@@ -242,4 +234,35 @@ public function processDataForm(array $params): string
             $params['dob_day'],
         );
     }
+
+    /**
+     * @param string $validDocument
+     * @param string $uuid
+     * @param array $templates
+     * @return mixed
+     */
+    public function fraudCheck(string $validDocument, string $uuid, array $templates): mixed
+    {
+        if ($validDocument === 'PASS') {
+            $fraudCheck = $this->opgApiService->requestFraudCheck($uuid);
+
+            if (
+                $fraudCheck['decision'] === 'ACCEPT' ||
+                $fraudCheck['decision'] === 'CONTINUE' ||
+                $fraudCheck['decision'] === 'REFER'
+            ) {
+                $template = $templates['success'];
+            } elseif ($fraudCheck['decision'] === 'NODECISION') {
+                $template = $templates['thin_file'];
+            } elseif ($fraudCheck['decision'] === 'STOP') {
+                $template = $templates['fraud'];
```

Comment:
Per the ticket, this should throw an exception because we haven't decided what we're doing with STOP responses yet.